pkg/repository/application_repository.go

```diff
@@ -25,6 +26,32 @@ type AppRepository struct {
 	Db *sqlx.DB
 }
 
+// FindAllByDAStatus implements AppRepositoryInterface.
+func (a *AppRepository) FindAllByDAStatus(ctx context.Context, da model.DataAvailabilitySelector, state model.ApplicationState) ([]model.Application, error) {
+	query := `SELECT *
+	FROM application
+	WHERE data_availability = decode($1, 'hex') AND state = $2`
```

Comment:
We may need to find all starting with the selector, as VF is going to save all the configuration.

Reply:
It's a bit hard use 'LIKE' with bytea. I'll investigating later.